packages/lib/src/components/Dropin/elements/filters.ts

```diff
@@ -1,5 +1,5 @@
 import promiseTimeout from '../../../utils/promiseTimeout';
-import { UIElement } from '../../../types';
+import { PaymentMethod, type StoredPaymentMethod, UIElement } from '../../../types';
```

Comment:
I am not entirely sure the difference, but usually the import types is used this way:

```
import types { PaymentMethod, StoredPaymentMethod, UIElement } from '../../../types';
```

Reply:
I think it's indeed a mistake, the Webstorm auto imported it. I'll fix it.